llvm/docs/ReleaseNotes.rst

```diff
@@ -238,6 +238,10 @@ Changes to the LLVM tools
   documented in `--help` output and the command guide. (`#90474
   <https://github.com/llvm/llvm-project/pull/90474>`)
 
+* llvm-readobj's LLVM output has changed format for ELF core files. The
```

Comment:
Sorry, should have noticed this earlier - you might as well mention the bug fix at the same time. Also, if you look at the grammar of other release notes, you'll notice this note isn't quite consistent.
```suggestion
* llvm-readobj's LLVM output format for ELF core files has been changed. Similarly, the JSON format has been fixed for this case. The
```
(make sure to reflow as necessary at the line limit)

Reply:
done